apps/backend/config/plugins.js

```diff
@@ -0,0 +1,11 @@
+module.exports = () => ({
+  upload: {
+    config: {
+      provider: "local",
+      providerOptions: {
+        // 250MB
+        sizeLimit: 250000000,
+      },
```

Comment:
```suggestion
      // 250MB
      sizeLimit: 250 * 1024 * 1024,
```
`providerOptions` is deprecated and the noumber of bytes wasn't quite right.